addons/html_editor/static/src/others/chatgpt_alternatives_dialog.js

```diff
@@ -0,0 +1,112 @@
+import { useState, status } from "@odoo/owl";
+import { ChatGPTDialog } from "./chatgpt_dialog";
+
+export const DEFAULT_ALTERNATIVES_MODES = {
+    correct: 'Correct',
```

Comment:
shouldn't this be translated? (import `_t`)

Reply:
I believe you're absolutely correct ^^